src/iarc7_motion/intermediary_state.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+class IntermediaryState(object):
```

Comment:
Can we rename this to TransitionData? This can be thought of as the intermediary state because it is all the data associated with a state. But more specifically it is the data and I think that in order to be called a state it should be a providing a summary of the data.

Also the word used to describe it everywhere else is transition thus leading me to TransitionData.

The current name also doesn't play well with how the statemanager actually handles the state transition checking making the user wonder why the task needs to know the intermediary state. The answer is it doesn't but it does need to know data relevant to transitioning.

Reply:
I agree, I like that name better. will fix.

Reply:
fixed